src/components/new-request/search.vue

```diff
@@ -47,20 +47,19 @@
             @business="onBusiness($event)"
           />
         </template>
-        <v-text-field
-          v-else
-          append-icon="mdi-close"
-          readonly
-          filled
-          :label="businessLookupLabel"
-          :value="business.legalName"
-          :rules="rules"
-          :hint="businessLookupHint"
-          persistent-hint
-          autofocus
-          @click:append="onBusiness(null)"
-          @keyup.delete="onBusiness(null)"
-        />
+        <v-form v-else ref="businessLookupTextFieldForm">
```

Comment:
I wrapped the text field in a v-form so that I can call the `validate()` function of the ref.

Reply:
You can call `validate()` on the component itself.

FYI, forms catch events (like Enter or Escape) and do something with that (like reload the page). If you use a form, test thoroughly to avoid unwanted behaviours.

Reply:
It didn't work. I tried calling the function on the v-text-field itself, the text field wasn't triggering the validation.

On it. I'll test thoroughly.

Reply:
Weird it didn't work.

OK, leaving it with you. Form is OK if that works fine. OK to resolve.

Reply:
I tested thoroughly, everything looks good Sev! 🙌